tests/php/Controllers/SiteTreeTest_NamespaceMapTestNodeController.php

```diff
@@ -5,7 +5,7 @@
 use SilverStripe\CMS\Controllers\ContentController;
 use SilverStripe\Dev\TestOnly;
 
-class SiteTreeTest_NamespaceMapTestController extends ContentController implements TestOnly
+class SiteTreeTest_NamespaceMapTestNodeController extends ContentController implements TestOnly
```

Comment:
This was renamed to match the corresponding SiteTreeTest_NamespaceMapTestNode which was also renamed